app/src/main/java/com/example/trempelapp/data_layer/inMemory/SharedPreferencesManagerImpl.kt

```diff
@@ -1,8 +1,23 @@
 package com.example.trempelapp.data_layer.inMemory
 
-class SharedPreferencesManagerImpl : SharedPreferencesManager {
+import android.content.Context
+import android.content.SharedPreferences
+import android.util.Log
+import com.example.trempelapp.utils.TOKEN_PREFERENCES_FILE
+import io.reactivex.Single
+import javax.inject.Inject
 
-    override fun getToken() {
-        TODO("Not yet implemented")
+class SharedPreferencesManagerImpl @Inject constructor(context: Context) :
```

Comment:
do not store context here, as you are storing SharedPreferences here

Reply:
done